### PR TITLE
cpe_bluetooth resource change

### PR DESCRIPTION
**Pull request by erikng**
Based on what @mikedodge04 said, here is a stab at cpe_bluetooth.

If this looks good, I'll keep going.

### CODE REVIEW

**Comment by cgerke** [comment]
+cpe_bluetooth 'Apply blueooth profile'  << TYPO

**Comment by erikng** [comment]
Thanks.

File: chef/cookbooks/cpe_bluetooth/resources/cpe_bluetooth.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+#
+
+resource_name :cpe_bluetooth
+default_action :run
+
+bt_prefs = {}
+
+action :run do
+  bt_prefs = node['cpe_bluetooth'].reject { |_k, v| v.nil? }
+  unless bt_prefs.empty?
```
Comment (mikedodge04) [inline chef/cookbooks/cpe_bluetooth/resources/cpe_bluetooth.rb]: return if bt_prefs.empty?

**Comment by erikng** [comment]
@mikedodge04 not seeing the comma, but the return if is now done.

**Pull request closed without merging**